server/controllers/users.js

```diff
@@ -0,0 +1,87 @@
+import generateRandomString from "crypto-random-string";
+import database from "../models/index";
+import { getBaseUrl, httpResponse, serverError } from "../helpers/index";
+import { sendResetEmail } from "../emails/email";
+import userResponse from "../helpers/userResponse";
+
+const User = database["Users"];
+
+export const sendPasswordReset = async (req, res, next) => {
+  const userEmail = req.body.email;
+  try {
+    const verifiedUser = await User.findOne({
+      where: {
+        email: userEmail
+      }
+    });
+    if (!verifiedUser || Object.keys(verifiedUser).length === 0) {
+      return httpResponse(res, {
+        statusCode: 404,
+        message: "Invalid Email"
+      });
+    }
+    const {
+      dataValues: { id, firstName, lastName, email }
+    } = verifiedUser;
+    const userResetToken = generateRandomString(64);
+    const resetUrl = getBaseUrl(req)
+      .concat("/users/resetPassword/")
+      .concat(`${id}/`)
+      .concat(userResetToken);
```

Comment:
Can't we use the normal `template string` to achieve the same purpose instead of `concat`?